airbyte-integrations/connectors-performance/destination-harness/src/main/java/io/airbyte/integrations/destination_performance/Main.java

```diff
@@ -97,6 +109,27 @@ public static void main(final String[] args) {
     System.exit(0);
   }
 
+  /**
+   * Modifies the catalog in place to update the syncMode to INCREMENTAL | APPEND to match CDC
+   * {@link CdcMssqlSourceAcceptanceTest} for reference. If the syncMode isn't INCREMENTAL then no-op
+   * since default catalog is FULL_REFERESH
+   *
+   * @param catalog ConfiguredCatalog to be modified
+   * @param syncMode syncMode to update to
+   */
+  @VisibleForTesting
+  static void updateSyncMode(final JsonNode catalog, final String syncMode) {
```

Comment:
Would this work on any of the streams in destination?
As an alternative I thought of adding a new catalog with incremental sync mode and have it as a separate dataset. e.g `10m_incremental`.

Reply:
This would work on any streams within a destination. As long as the `ConfiguredCatalog` is of the correct format the destination knows how to process this contract to understand the _way_ to write data within the destination